app/controllers/application_letters_controller.rb

```diff
@@ -57,6 +57,6 @@ def set_application
 
     # Only allow a trusted parameter "white list" through.
     def application_params
-      params.require(:application_letter).permit(:motivation, :event_id, :status)
+      params.require(:application_letter).permit(:motivation, :user_id, :event_id, :status)
```

Comment:
do you know why it was removed? An application letter without an user seems not really useful to me